### PR TITLE
Use value from setting to determine browser state

### LINKED ISSUE #172: Enabling Session Management middleware yields KeyError on `op_browser_state`
We've been using this project for a while for our OpenID Connect setup. Recently we've been looking into Session Management and have enabled this. After enabling it, we're seeing a lot of `KeyError`s on the [`op_browser_state` cookie](https://github.com/juanifioren/django-oidc-provider/blob/53166fb9eaa4bf5a1f5a314b4f83e848da0b0e75/oidc_provider/lib/endpoints/authorize.py#L200).
We're able to reproduce this by having a client that already has an active sessioncookie without Session Management, and then enabling it. `create_response_uri` blindly assumes that the cookie should be there, while it won't be set until the [first response](https://github.com/juanifioren/django-oidc-provider/blob/f24445e4213646ef24154878c15bc2d2d752454a/oidc_provider/middleware.py#L14).

I'd be happy to make a PR for this, but I'm not sure what the correct course of action should be in `create_response_uri`.

### PR DESCRIPTION
**Pull request by gertjanol**
@wojtek-fliposports 

Fixes #172

### CODE REVIEW

File: oidc_provider/settings.py [hunk 1]
```diff
@@ -75,6 +80,18 @@ def OIDC_SESSION_MANAGEMENT_ENABLE(self):
         return False
 
     @property
+    def OIDC_UNAUTHENTICATED_SESSION_MANAGEMENT_KEY(self):
+        """
+        OPTIONAL. Supply a fixed string to use as browser-state key for unauthenticated clients.
+        """
+
+        # Memoize generated value
```
Comment (kerrermanisNL) [inline oidc_provider/settings.py]: *Nitpick: memorize :)

**Comment by wojtek-fliposports** [comment]
Great piece of contribution. Thank You :)

**Comment by gertjanol** [comment]
Your welcome! Do you have an estimate when this will be released?

**Comment by wojtek-fliposports** [comment]
I hope this week. I must prepare changelog and perform some tests on develop branch and if merging pass all I will poke @juanifioren to release it.
I think we can mark this release as Security Update to speed this up.

**Pull request merged**